torchvision/prototype/transforms/_misc.py

```diff
@@ -20,7 +20,7 @@ class Lambda(Transform):
     def __init__(self, fn: Callable[[Any], Any], *types: Type):
         super().__init__()
         self.fn = fn
-        self.types = types
+        self.types = types or (object,)
```

Comment:
This is perfectly valid. We don't use this often in the codebase.

A possible alternative and could be something along the lines of:
```python
    def __init__(self, fn: Callable[[Any], Any], types: Tuple[Type, ...] = (object,)):
        super().__init__()
        self.fn = fn
        self.types = types
```

Just putting it other there, we don't have to do this.